### PR TITLE
add *basic* js functionality for translation api.

### PR DESCRIPTION
**Pull request by bosticc**
Super basic for right now, once we get this working with the servlet will more interactive

### CODE REVIEW

**Comment by bosticc** [comment]
*be more interactive

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -10,4 +10,23 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License
\ No newline at end of file
+// limitations under the License
+function requestTranslation() {
+    const text = document.getElementById('text').value;
```
Comment (jdstriker731) [inline src/main/webapp/script.js]: A suggestion Cam or Zu made to me once was putting calls to document.getElementById() outside of any function call (i.e. making them global variables). This way, you only traverse the DOM one time to lookup these elements, as opposed to every time your function runs.

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -10,4 +10,23 @@
...
+    const languageCode = document.getElementById('language').value;
```
Comment (jdstriker731) [inline src/main/webapp/script.js]: Also, the only other thing is the whitespace. It should be two "spaces" rather than four

**Review by jdstriker731 — CHANGES_REQUESTED** [review]

File: src/main/webapp/script.js (hunk 1 above)
Comment (bosticc) [reply src/main/webapp/script.js]: I'd want the function to get the text value every time it runs though, cuz that means that someone is making a new request to get their stuff translated no?

Comment (jdstriker731) [reply src/main/webapp/script.js]: Ohh! Yeah, then I think you're fine with this.

**Review by jdstriker731 — CHANGES_REQUESTED** [review]

File: src/main/webapp/script.js (hunk 2 above)
Comment (bosticc) [reply src/main/webapp/script.js]: gotcha

**Comment by jdstriker731** [comment]
LGTM!

**Review by jdstriker731 — APPROVED** [review]

**Pull request merged**